shared/locales/download/download.en-US.properties

```diff
@@ -66,11 +66,6 @@ file_open_error_download_right_button          = Delete
 restart_download_error                         = There was an error while restarting the download. Please try to download again.
 no-downloads                                   = No downloads
 # File sizes
-byteUnit-B                    = B
-byteUnit-KB                   = KB
-byteUnit-MB                   = MB
-byteUnit-GB                   = GB
-byteUnit-TB                   = TB
 fileSize                      = {{size}} {{unit}}
```

Comment:
Let’s share this line (`fileSize`) as well, please. It will make a lot more sense for our l10n contributors if this string lives in the same file as the localized data units.